generator/src/main/resources/template/event-template.mustache

```diff
@@ -112,6 +112,15 @@ public class {{capitalizedSubject}}{{capitalizedPredicate}}CDEvent extends {{cap
         return "{{schemaFileName}}";
     }
 
+    /**
+    *
+    * @return custom schema URI
+    */
+    @Override
+    public URI customSchemaUri() {
```

Comment:
NIT: the field name is `schemaUri` - perhaps the method name should match it for clarity?

Reply:
changing it to `contextSchemaUri()`